code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -72,6 +72,25 @@ Contains:
 	armor = list(melee = 40, bullet = 30, laser = 30,energy = 30, bomb = 50, bio = 90, rad = 20)
 	strip_delay = 120
 
+/obj/item/clothing/suit/space/swat/equipped(mob/user, slot)
+	. = ..()
+	START_PROCESSING(SSobj, src)
+
+/obj/item/clothing/suit/space/swat/process()
+	if(ishuman(loc))
+		var/mob/living/carbon/human/hut = loc
+		hut.say("HUT HUT HUT")
+		if(prob(25))
+			playsound(src, 'sound/misc/huthuthut.ogg', 50, 1)
+
+/obj/item/clothing/suit/space/swat/dropped(mob/user)
+	. = ..()
+	STOP_PROCESSING(SSobj, src)
+
+/obj/item/clothing/suit/space/swat/Destroy()

```

Comment:
if the mob is deleted directly (like by admins or gib()) this will get deleted before dropped is called.